specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-09-01-preview/apimanagement.json

```diff
@@ -364,7 +364,18 @@
       "type": "string",
       "description": "The name of the API Management gateway.",
       "minLength": 1,
-      "maxLength": 50,
+      "maxLength": 45,
+      "pattern": "^[a-zA-Z](?:[a-zA-Z0-9-]*[a-zA-Z0-9])?$",
+      "x-ms-parameter-location": "method"
+    },
+    "ConfigConnectionIdParameter": {
+      "name": "configConnectionId",
```

Comment:
>configConnectionId [](http://example.com/codeflow?start=15&length=18)

Please be consistent between "name" and "Id". If it's a name, it's chosen by the client according to whatever convention they want. If it's an Id, it needs explanation in the description about the format and type of Id it is (e.g. integer, GUID, resourceId, etc.). This looks more like a "name" to me. If that's correct, it should have "Name" in the name instead of "Id".